doc/buildWithDocker.md

```diff
@@ -19,44 +20,38 @@ This example will build the firmware, generate the MCUBoot image and generate th
 
 ```bash
 cd <project_root> # e.g. cd ./work/Pinetime
-docker run --rm -it -v $(pwd):/sources infinitime-build
+docker run --rm -it -v $(pwd):/sources --user $(id -u):$(id -g) infinitime-build
 ```
 
+By default, the container runs as `root`, which is not convenient as all the file generated by the build will also belong to `root`. The parameter `--user` overrides that default behavior. The command above will run as your current user. 
+
 If you only want to build a single CMake target, you can pass it in as the first parameter to the build script. This means calling the script explicitly as it will override the `CMD`. Here's an example For `pinetime-app`:
 
 ```bash
-docker run --rm -it -v $(pwd):/sources infinitime-build /opt/build.sh pinetime-app
+docker run --rm -it -v $(pwd):/sources --user $(id -u):$(id -g) infinitime-build /opt/build.sh pinetime-app
 ```
 
-The image is built using 1000:1000 for the user id and group id. If this is different to your user or group ids (run `id -u` and `id -g` to find out what your id values are if you are unsure), you will need to override them via the `--user` parameter in order to prevent permission errors with the output files (and the cmake build cache).
-
-Running with this image is the same as above, you just specify the ids to `docker run`:
+## Using the image from Docker Hub
 
-```bash
-docker run --rm -it -v $(pwd):/sources --user $(id -u):$(id -g) infinitime-build
-```
+The image is available via Docker Hub for both the amd64 and arm64v8 architectures at [infinitime/infinitime-build](https://hub.docker.com/repository/docker/infinitime/infinitime-build).
 
-Or you can specify your user id and group id (by number, not by name) directly:
+It can be pulled (downloaded) using the following command:
 
 ```bash
-docker run --rm -it -v $(pwd):/sources --user 1234:1234 infinitime-build
+docker pull infinitime/infinitime-build
 ```
 
-## Using the image from Docker Hub
-
-The image is available via Docker Hub for both the amd64 and arm64v8 architectures at [jf002/infinitime-build](https://hub.docker.com/repository/docker/jf002/infinitime-build).
+The command line to build the project is the same as above. You just need to change the image name to `infinitime/infinitime-build`:
 
-It can be pulled (downloaded) using the following command:
-
-```bash
-docker pull jf002/infinitime-build
+```
+docker run --rm -it -v $(pwd):/sources --user $(id -u):$(id -g) infinitime/infinitime-build
 ```
```

Comment:
the mentioned image name we need to replace is exactly the same we use in the command above. I think this paragraph is obsolete